src/cls/tabular/cls_tabular.cc

```diff
@@ -1423,7 +1367,6 @@ int exec_query_op(cls_method_context_t hctx, bufferlist *in, bufferlist *out)
                     // call associated process method based on ds type
                     switch (meta.blob_format) {
 
-
                     case SFT_JSON: {
 
                         sky_root root = \
```

Comment:
This can be removed from JSON case also, but I will assign to the person working JSON.